src/transformers/tokenization_utils_base.py

```diff
@@ -724,18 +724,18 @@ def convert_to_tensors(
                     #     tensor = tensor[None, :]
 
                     self[key] = tensor
-            except:  # noqa E722
+            except Exception as e:  # noqa E722
```

Comment:
```suggestion
            except Exception as e:
```
I don't think you'll need the quality exception anymore